Makefile

```diff
@@ -25,46 +25,27 @@ APPNAME      = "Boilerplate"
 APPVERSION_M = 1
 APPVERSION_N = 0
 APPVERSION_P = 1
-APPVERSION   = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 
```

Comment:
Could be the other way, by setting APPVERSION and splitting after to generate the different parts. 
Anyway this doesn't improve a lot, so might not be worth the obfuscation?